src/Item_Project.php

```diff
@@ -236,7 +240,7 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
                    // Not used now
                     if (
                         Session::haveRight("project", Project::READALL)
-                        && ($item instanceof CommonDBTM)
+                        && ($item instanceof CommonDBTM && in_array($item->getType(), $CFG_GLPI["project_asset_types"]))
```

Comment:
Some parenthesis are useless here.

```suggestion
                        && $item instanceof CommonDBTM
                        && in_array($item->getType(), $CFG_GLPI["project_asset_types"])
```